crates/stc_ts_type_checker/tests/conformance/jsx/tsxSpreadAttributesResolution7.tsx

```diff
@@ -2,7 +2,7 @@
 // @jsx: preserve
 // @noLib: true
 // @skipLibCheck: true
-// @libFiles: react.d.ts,lib.d.ts
+// @libFiles: lib.d.ts,react.d.ts
```

Comment:
Had to add this for testing, checker fails if `react.d.ts` was loaded before `lib.d.ts`

Reply:
As a simple workaround, I can just `sort()` the lib_files, but I don't know if it's desired.

Reply:
I think we should never change the test case in the conformance test suite, as those are copied from `tsc`.

Reply:
Understood. I'll remove these changes.